src/styles/index.scss

```diff
@@ -1,3 +1,95 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
```

Comment:
The font-family declaration should list the generic family name 'sans-serif' last, after specific font names like 'Roboto' and 'Arial'. This ensures that if none of the specified fonts are available, a generic sans-serif font will be used.